src/Mailgun/Mailgun.php

```diff
@@ -208,4 +274,29 @@ public function BatchMessage($workingDomain, $autoSend = true)
     {
         return new BatchMessage($this->restClient, $workingDomain, $autoSend);
     }
+
+    /**
+     * @param string $messageUrl
+     *
+     * @return \stdClass
+     */
+    public function getMessageFromUrl($messageUrl)
+    {
+        if ($messageUrl === null) {
+            return null;
```

Comment:
According to PSR standards, when we `return null;` we can just write `return;`

Also, when we return `null` is must be declared on `@return` annotation too, so `@return \stdClass|null`